packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -930,6 +989,90 @@ export function CreateTaskForm({
+                  </Grid>
+                  <Grid item xs={6}>
+                    {dispatchType === DispatchType.Fleet ? (
+                      <TextField
+                        select
+                        id="dispatch-fleet-type"
+                        label="Select fleet"
+                        variant="outlined"
+                        fullWidth
+                        margin="normal"
+                        value={taskRequest.fleet_name ?? ''}
+                        defaultValue=""
+                        onChange={handleDispatchFleetTargetChange}
+                      >
+                        {fleets &&
+                          Object.keys(fleets).map((fleetName) => {
+                            return (
+                              <MenuItem value={fleetName} key={fleetName}>
+                                {fleetName}
+                              </MenuItem>
+                            );
+                          })}
+                      </TextField>
+                    ) : dispatchType === DispatchType.Robot ? (
+                      <TextField
+                        select
+                        id="dispatch-robot-type"
+                        label="Select robot"
+                        variant="outlined"
+                        fullWidth
+                        margin="normal"
+                        value={robotDispatchTarget?.robot ?? ''}
+                        defaultValue=""
+                        onChange={handleDispatchRobotTargetChange}
+                      >
+                        {fleets &&
+                          Object.keys(fleets).flatMap((fleetName) => {
+                            const fleetRobots = [
+                              <Divider key={`${fleetName}_divider`} />,
+                              <MenuItem value={fleetName} key={fleetName} disabled>
+                                {fleetName}
+                              </MenuItem>,
+                            ];
+                            return fleetRobots.concat(
+                              fleets[fleetName].map((robotName) => (
+                                <MenuItem value={robotName} key={robotName}>
+                                  {robotName}
+                                </MenuItem>
+                              )),
+                            );
+                          })}
+                      </TextField>
+                    ) : (
+                      <TextField
+                        disabled
+                        id="dispatch-automatic-type"
+                        variant="outlined"
+                        fullWidth
+                        margin="normal"
+                      />
+                    )}
+                  </Grid>
+                </Grid>
+                <Divider
+                  orientation="horizontal"
+                  flexItem
+                  style={{ marginTop: theme.spacing(2), marginBottom: theme.spacing(2) }}
```

Comment:
Use `sx` instead of `style`

Reply:
https://github.com/open-rmf/rmf-web/pull/1004/commits/cdf7fbaef4c5ee34d37ba2fce715425412da2c39